dennis/minisix.py

```diff
@@ -10,5 +10,12 @@
 
 else:
     textclass = str
-    from html.parser import HTMLParser, HTMLParseError  # noqa
+    from html.parser import HTMLParser  # noqa
     from itertools import zip_longest as izip_longest  # noqa
+    try:
+        from html.parser import HTMLParseError  # noqa
+    except:
```

Comment:
This should probably be an `ImportError` or something like that and not a bare except.